idd/Energy+.idd.in

```diff
@@ -64711,12 +64896,11 @@ WaterHeater:Sizing,
        \note only used if for WaterHeater:Stratified
        \minimum 0.0
 
-WaterHeater:HeatPump,
-       \min-fields 31
+WaterHeater:HeatPump:PumpedCondenser,
+       \min-fields 29
        \memo This object models an air-source heat pump for water heating.
```

Comment:
@nmerket The memos for these two waterheater:heatpump objects should have a sentence explaining the difference.  Hindsight (not suggesting we change this now) I only see about three fields that are different between these two objects, so we could have kept this a WaterHeater:HeatPump and added a few new fields.  Also, the condenser inlet/outlet node fields could be eliminated and mined from the coil object (also not suggesting you change that now).
@Myoldmopar At the risk of opening a huge debate, I'm going to post a new issue to discuss and clarify object design conventions and make sure we get that documented.  See #4995